interview-questions.md

```diff
@@ -0,0 +1,79 @@
+# ASSESSMENT 2: Interview Practice Questions
+
+Answer the following questions.
+
+First, without external resources. Challenge yourself to answer from memory as if you were in a job interview.
+
+Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
+
+1. What is the difference between a parameter and an argument?
+
+Your answer: Parameters are invoke withing a function while  
+arguments are invoke by functions. lol
+
+functions take argument
+functions perform actions on parameters
+
+(my answer after wrapping my head around it.)
```

Comment:
This is okay too! Love that you included this because it just proves that in a real interview you would have talked yourself in the right direction or if you had time to yourself you could wrap your head around it. Great job!